daemon/src/projection.rs

```diff
@@ -858,27 +879,9 @@ impl xtra::Actor for Actor {
     type Stop = ();
     async fn started(&mut self, ctx: &mut xtra::Context<Self>) {
         let this = ctx.address().expect("we just started");
-        let pool = self.db.clone();
-
-        self.tasks.add_fallible(
-            {
-                let this = this.clone();
-
-                async move {
-                    let mut conn = pool.acquire().await?;
-                    let vec = db::load_all_cfd_ids(&mut conn).await?;
-
-                    for id in vec {
-                        let _: Result<(), xtra::Disconnected> = this.send(CfdChanged(id)).await;
-                    }
-
-                    anyhow::Ok(())
-                }
-            },
-            |e| async move {
-                tracing::warn!("Failed to do initial rehydratation of CFDs: {e:#}");
-            },
-        );
+        this.send_async_safe(Initialize)
```

Comment:
❓ What's the logic behind using `send_async_safe` here? Can we run into trouble by letting the queue of `CfdChanged` messages grow whilst we process the `Initialize` message?

Reply:
Theoretically yes and that *could* lead to failures if e.g. takers trigger a lot of rollovers when we start up the maker (which can happen sometimes, if the maker was down for longer or we start right after the full hour), but that should not cause problems. 

Requests would be piling up until the initial loading of the projection is done. Note, that those requests piling up is not really an issue. If the maker does not respond to the taker in time the taker's rollover protocol will just timeout eventually. 
Same for contract setup, and same for collab settlement - it might just gracefully fail on the taker side if the maker does not respond because it is still starting up and is thus not accepting protocol requests.

Of course this cannot scale forever, and our storage and loading performance optimizations should make it in soon. But we are already working on that so I don't see blockers.